script/bigscreenplayer.js

```diff
@@ -142,13 +145,24 @@ define('bigscreenplayer/bigscreenplayer',
         });
       }
 
+      function setSubtitlesEnabled (value) {
+        playerComponent.setSubtitlesEnabled(value);
+        callSubtitlesCallbacks(value);
+      }
+
+      function isSubtitlesEnabled () {
+        return playerComponent ? playerComponent.isSubtitlesEnabled() : false;
+      }
+
       return {
-        init: function (playbackElement, bigscreenPlayerData, newWindowType, enableSubtitles, callbacks) {
+        init: function (newPlaybackElement, bigscreenPlayerData, newWindowType, enableSubtitles, callbacks) {
           Chronicle.init();
+          resizer = Resizer();
           DebugTool.setRootElement(playbackElement);
           DebugTool.keyValue({key: 'framework-version', value: Version});
           windowType = newWindowType;
           serverDate = bigscreenPlayerData.serverDate;
+          playbackElement = newPlaybackElement;
```

Comment:
Instead of storing a reference to element can we not get it from the DOM when we require it?

Reply:
Would rather store in this instance. We take the whole playback element, so we'd need to know the id of this. It's not the video element that is resized, it's the whole containing div.